backend/src/middleware/sluggifyMiddleware.js

```diff
@@ -25,9 +25,5 @@ export default {
       args.slug = args.slug || (await uniqueSlug(args.title, isUniqueFor(context, 'Post')))
       return resolve(root, args, context, info)
     },
-    CreateCategory: async (resolve, root, args, context, info) => {
-      args.slug = args.slug || (await uniqueSlug(args.name, isUniqueFor(context, 'Category')))
-      return resolve(root, args, context, info)
-    },
```

Comment:
good cleanup... we shouldn't need to create categories, and if we do, it would be an admin request later